azerothcore/azerothcore-wotlk
# fix(DB/Creature): Fixed various static Vilebranch Trolls

## Changes Proposed:
- Added wandering to the Trolls where it makes sense, the Ooze and the Wolf Pup

## Issues Addressed:

- Closes #6824

## SOURCE:


## Tests Performed:

- Checked the DB after running the queries for any Trolls, Ooze or Wolf Pups missing movement. For those that did, I went to them in game to check whether it made sense that they were non-moving.


## How to Test the Changes:


DB:
1. Check the IDs 2640, 2642, 2643, 2644, 2645, 2646, 2647 and 2648 for any missing movement
2. Check the GUIDs 93735, 49288 and 93627 for any missing movement
3. On missing movement, verify that it makes sense

In Game:
1. Go to each mentioned GUID in the issue
2. Observe movement
3. On no movement, verify that it makes sense

## Known Issues and TODO List:


- It might make sense to add patrols to some of the trolls, but I don't know how to do that. Or, for that matter, whether there should be patrols.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
